spec/requests/api/v1/sessions/facebook_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'rails_helper'
+
+# Test suite for POST /api/v1/auth/facebook
+describe 'POST /api/v1/auth/facebook', type: :request do
```

Comment:
The tests are ok 👍 but I guess that you can add some tests to validate that the facebook sign_in is working properly. 
i.e: if the user doesn't exist, check that is being created or that is not being created twice if the user is already registered. Also you can check that the information is being assigned correctly.